pkg/scanners/cloudformation/parser/parameter.go

```diff
@@ -57,3 +60,67 @@ func (p *Parameter) UpdateDefault(inVal interface{}) {
 		p.inner.Default = passedVal
 	}
 }
+
+type Parameters map[string]any
+
+func (p *Parameters) Merge(other Parameters) {
+	for k, v := range other {
+		(*p)[k] = v
+	}
+}
+
+func (p *Parameters) UnmarshalJSON(data []byte) error {
```

Comment:
What about YAML? Should we add support for that too?

Reply:
That's right, AWS only supports json, so users will use json as well.